packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -72,6 +72,21 @@ export class FocusTrapZone extends React.Component<IFocusTrapZoneProps, {}> impl
     ) {
       this._returnFocusToInitiator();
     }
+
+    // Dispose of event handlers so their closures can be garbage-collected
+    if (this._disposeClickHandler) {
```

Comment:
Should you get rid of root, firstBumper, and lastBumper too, or is removing DOM node references from refs handled automatically?

Reply:
I haven't looked at React's source, but the fields utilizing React's Ref seem to be cleaned-up automatically. I considered your suggestion, but have not seen any of those refs in the heap snapshots. More motivation to use Refs wherever possible vs. manual property assignment.

Reply:
Interesting, that's good to know!